### PR TITLE
Debian Improvements

### PR DESCRIPTION
**Pull request by Anuskuss**
Continuation of #698.

## Summary by CodeRabbit

- **New Features**
	- Introduced new configuration file structure for `peerbanhelper`, enhancing organization.
	- Added a new `postrm` script for improved package removal processes.
	- New `prerm` script to manage service stopping before package removal.

- **Improvements**
	- Enhanced error handling and user management in the installation process.
	- Updated service configuration for better operational management and logging.

- **Bug Fixes**
	- Fixed user migration issues for versions prior to 7.1.2. 

- **Documentation**
	- Minor formatting updates in the `profile.yml` configuration file.

### CODE REVIEW

File: pkg/deb/usr/lib/systemd/system/peerbanhelper.service [hunk 1]
```diff
@@ -1,11 +1,14 @@
 [Unit]
 Description=PeerBanHelper Service
 After=network.target
+
 [Service]
 User=peerbanhelper
-WorkingDirectory=/etc/peerbanhelper
-ExecStart=/usr/bin/java -Dpbh.release=debian -Dpbh.datadir=/etc/peerbanhelper -Xmx512M -XX:+UseG1GC -XX:+UseStringDeduplication -XX:+ShrinkHeapInSteps -jar /usr/lib/peerbanhelper/PeerBanHelper.jar
+WorkingDirectory=/usr/lib/peerbanhelper
```
Comment (Gaojianli) [inline pkg/deb/usr/lib/systemd/system/peerbanhelper.service:7]: I think the wd should be the `$HOME` of user `peerbanhelper`, which means `/var/lib/peerbanhelper`. Anyway, it doesn't matter

**Comment by Gaojianli** [comment]
Thanks for your effort!

**Comment by Gaojianli** [comment]
It seems the ci failed on the `dev` branch, can you submit a fix? @Anuskuss 
image

**Comment by Anuskuss** [comment]
I'm not sure it's possible to change file permissions from GitHub's Web UI. Can you do that yourself?
```
git update-index --chmod=+x pkg/deb/DEBIAN/postrm pkg/deb/DEBIAN/preinst pkg/deb/DEBIAN/prerm
````

**Pull request merged**